cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -50,11 +50,10 @@
      {{- ' id="' ~ value.heading.text|slugify ~ '" ' if value.heading.text else ' ' -}}>
 
 {% if value.heading and value.heading.text %}
-    <{{ value.heading.level -}}
-        {%- if value.heading.icon -%}
-            {{- ' class="cf-icon cf-icon-' ~ value.heading.icon ~ '"' -}}
-        {%- endif -%}
-    >
+    <{{ value.heading.level }}>
+        {% if value.heading.icon -%}
+            {{ svg_icon(value.heading.icon) }}
+        {%- endif %}
```

Comment:
Suuuper nitpicky, but I think we want this to be `{% endif -%}` so there is no empty line if the condition is false. (Will also apply to the heading block template.)

Reply:
Wouldn't doing that remove the whitespace entirely and break the built in spacing if there is an icon?